src/amo/sagas/reviews.js

```diff
@@ -341,6 +350,14 @@ function* manageAddonReview(
       yield put(
         fetchAddon({ errorHandler, slug: reviewFromResponse.addon.slug }),
       );
+
+      yield put(
+        updateRatingCounts({
+          addonId: reviewFromResponse.addon.id,
+          oldReview,
+          newReview: createInternalReview(reviewFromResponse),
```

Comment:
So, we usually do that in the reducer. Why doing this differently here?

I also think the reducer could retrieve the `oldReview` _via_ `selectReview` in the reducer if `reviewId` is supplied.

Reply:
I originally wanted to do it like that but in https://github.com/mozilla/addons-frontend/pull/7064 the `addons` reducer will need to look at the old and new review when it recalculates the overall rating average for the add-on. This logic makes sense for the `addons` reducer because the overall rating average comes from the add-on API.

Reply:
Ok, that works for me.